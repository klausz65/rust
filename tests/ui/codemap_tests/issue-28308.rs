//@revisions: edition2024 edition2021
//@[edition2024] edition:2024
//@[edition2024] compile-flags: -Z unstable-options
//@[edition2021] edition:2021
fn main() {
    assert!("foo");
    //[edition2024]~^ ERROR mismatched types
    //[edition2021]~^^ ERROR mismatched types
}
