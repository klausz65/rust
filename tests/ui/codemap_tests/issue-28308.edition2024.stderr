error[E0308]: mismatched types
  --> $DIR/issue-28308.rs:6:13
   |
LL |     assert!("foo");
   |             ^^^^^ expected `bool`, found `&str`

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0308`.
